modules/microshift-config-cli-manifests.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-cli-manifests_{context}"]
+= Using CLI tools and creating manifests
+
+Configure your {product-title} using the supported command line arguments and environment variables.
+
+== Supported command-line arguments and environment variables
+
+[cols="4",options="header"]
+|===
+|Field Name
+|CLI Argument
+|Environment Variable
+|Description
+
+|clusterCIDR
+|--cluster-cidr
+|MICROSHIFT_CLUSTER_CLUSTERCIDR
+|A block of IP addresses from which Pod IP addresses are allocated
```

Comment:
Should put the field name, CLI arg, and env var in backticks, and add a `.` at the end of the descriptions, e.g.:
```suggestion
|`clusterCIDR`
|`--cluster-cidr`
|`MICROSHIFT_CLUSTER_CLUSTERCIDR`
|A block of IP addresses from which Pod IP addresses are allocated.
```